### PR TITLE
docs: update audit-scanner explanation

### PR DESCRIPTION
**Pull request by fabriziosestito**
## Description

Fix: #365

### CODE REVIEW

File: docs/explanations/audit-scanner/audit-scanner.md [hunk 1]
```diff
@@ -20,10 +21,10 @@ The Audit Scanner feature is available starting from Kubewarden 1.7.0 release
 :::
 
 The `audit-scanner` component constantly checks resources in the cluster.
-It flags the ones that don't adhere with the Kubewarden policies deployed in the cluster.
+It flags the ones that don't adhere to the Kubewarden policies deployed in the cluster.
 
 Policies evolve over time.
-New ones are deployed, existing ones are updated.
+New ones are deployed, and existing ones are updated.
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/audit-scanner.md:27]: Why change?

Also, if the and is wanted then the ',' is probably not.

File: docs/explanations/audit-scanner/audit-scanner.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ sidebar_position: 50
 title: What is the Audit Scanner?
 description: An overview of the Kubewarden Audit Scanner.
 keywords: [kubewarden, audit scanner, kubernetes]
-doc-persona: [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
+doc-persona:
+  [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/audit-scanner.md:8]: Ok, fine. Is a formatter of some sort being used? We should all maybe be using the same one in that case?

File: docs/explanations/audit-scanner/audit-scanner.md [hunk 3]
```diff
@@ -20,10 +21,10 @@ The Audit Scanner feature is available starting from Kubewarden 1.7.0 release
 :::
 
 The `audit-scanner` component constantly checks resources in the cluster.
-It flags the ones that don't adhere with the Kubewarden policies deployed in the cluster.
+It flags the ones that don't adhere to the Kubewarden policies deployed in the cluster.
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/audit-scanner.md:24]: Yep, better.

File: docs/explanations/audit-scanner/audit-scanner.md (hunk 1 above)
Comment (jhkrug) [inline docs/explanations/audit-scanner/audit-scanner.md:27]: Why change?

Also, if the 'and' is wanted then the ',' is probably not.

File: docs/explanations/audit-scanner/audit-scanner.md [hunk 4]
```diff
@@ -76,7 +74,7 @@ See the policy authors [documentation](../../tutorials/writing-policies/index.md
 
 The audit scanner in Kubewarden requires specific Role Based Access Control (RBAC) configurations to be able to scan Kubernetes resources and save the results.
 A correct default Service Account with those permissions is created during the installation.
-The user can create and configure their own ServiceAccount to fine tune access to resources.
+The user can create and configure their own ServiceAccount to fine-tune access to resources.
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/audit-scanner.md:77]: Maybe?

```suggestion
The user can create their own ServiceAccount to configure access to resources.
```

Idiomatic English like 'fine-tune' is great, but it does rely on shared understanding that may not always be in place with an international audience.

In this case the audience need to understand it's to do with getting musical instruments to sound correct, or perhaps it refers to making sure that an engine is operating optimally, or some of the phrases many other uses. But 'fine-tune' __is__ widely used, so .....

File: docs/explanations/audit-scanner/policy-reports.md [hunk 5]
```diff
@@ -3,7 +3,8 @@ sidebar_label: Policy Reports
 title: Audit Scanner - Policy Reports
 description: The policy reports that the Audit Scanner produces.
 keywords: [kubewarden, kubernetes, audit scanner]
-doc-persona: [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
+doc-persona:
+  [kubewarden-user, kubewarden-operator, kubewarden-policy-developer, kubewarden-integrator]
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/policy-reports.md:7]: Ok, formatter again?

File: docs/explanations/audit-scanner/policy-reports.md [hunk 6]
```diff
@@ -43,140 +43,174 @@ The audit results generated by the scanner include:
...
   warn: 0
 ```
+
+## Policy Reporter UI
+
+The Policy Reporter is shipped as a subchart of `kubewarden-controller`.
+Refer to the [Audit Scanner Installation](../../howtos/audit-scanner)
+page for more information.
+
+The Policy Reporter UI provides a dashboard showcasing all violations
```
Comment (jhkrug) [inline docs/explanations/audit-scanner/policy-reports.md:201]:
```suggestion
The Policy Reporter UI provides a dashboard showing all violations
```

Questions about idiomatic English again. What's a 'showcase'. Would the audience always understand that? https://dictionary.cambridge.org/dictionary/english/showcase

**Review by jhkrug — COMMENTED** [review]
LGTM. I had a few minor observations. All good though.

We are probably getting to the point where we try to standardize a bit on tools and formatters. Prettier, Vale, Grammarly, LanguageTool, whatever.

**Review by jvanz — APPROVED** [review]

**Pull request merged**